cli/src/lib.rs

```diff
@@ -826,16 +844,32 @@ fn init(
     fs::write(".prettierignore", rust_template::prettier_ignore())?;
 
     // Build the program.
+    if force {
+        if solidity {
+            remove_dir_all(std::env::current_dir()?.join("solidity").join(&project_name))?;
+        } else {
+            remove_dir_all(std::env::current_dir()?.join("programs").join(&project_name))?;
+        }
+    }
+
```

Comment:
PR is looking good but noticed small comment mismatch here

Reply:
Fixed in https://github.com/coral-xyz/anchor/pull/2698/commits/e9bfc6a48a17eec65b31b5ca6a958f87a68871f0